core/src/language/python.rs

```diff
@@ -173,10 +193,23 @@ impl Language for Python {
         special_ty: &SpecialRustType,
         generic_types: &[String],
     ) -> Result<String, RustTypeFormatError> {
+        if let Some(mapped) = self.type_mappings.get(&special_ty.to_string()) {
+            let custom_translation_functions = json_translation_for_type(mapped);
+            if let Some(custom_translation_function) = custom_translation_functions {
+                self.custom_translation_functions
+                    .push(custom_translation_function.serialization_content);
+                self.custom_translation_functions
+                    .push(custom_translation_function.deserialization_content);
+            }
+            return Ok(mapped.to_owned());
+        }
         match special_ty {
-            SpecialRustType::Vec(rtype)
-            | SpecialRustType::Array(rtype, _)
-            | SpecialRustType::Slice(rtype) => {
+            SpecialRustType::Vec(rtype) => {
+                // TODO: https://github.com/1Password/typeshare/issues/231
+                self.add_import("typing".to_string(), "List".to_string());
+                Ok(format!("List[{}]", self.format_type(rtype, generic_types)?))
+            }
```

Comment:
This diff is now no longer needed right? It can have the same translation as Array and Slice